package.json

```diff
@@ -6,6 +6,7 @@
     "validate": "bin/validate.js"
   },
   "scripts": {
+    "prestart": "cp bin/pre-commit .git/hooks/ && chmod +x .git/hooks/pre-commit && echo 'hook copied'",
```

Comment:
There's no `start` nor is there a `server.js` so what's the point of `prestart`? Shouldn't this be `install-hooks` and shouldn't we mention in the README to run it?

Reply:
I will instead add install-hooks to `make deps`, makes it easier for users.